app/javascript/oldjs/controllers/network_router/network_router_form_controller.js

```diff
@@ -39,7 +39,6 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$scope', 'netwo
     } else {
       return API.get('/api/network_routers/' + networkRouterFormId + '?attributes=name,admin_state_up,cloud_network_id,cloud_tenant.name,ext_management_system.id,ext_management_system.name,extra_attributes').then(function(data) {
         Object.assign(vm.networkRouterModel, data);
-        vm.networkRouterModel.admin_state_up = vm.networkRouterModel.admin_state_up === 't' ? true : false;
```

Comment:
I didn't understand. Only this line id responsible for old form generation? I am expecting more lines to be deleted for old form.